### PR TITLE
Build Transaction: time bounds timestamp human readable date

### PR DESCRIPTION
**Pull request by quietbits**
image

image

### CODE REVIEW

**Comment by janewang** [comment]
I think it's better if it's UTC. 

> By international convention, UTC is equivalent to GMT, however their measurements differ: GMT is measured from midday, whereas UTC is measured from midnight. UTC is the legal basis of time in the world.

**Comment by quietbits** [comment]
> I think it's better if it's UTC.
> 
> > By international convention, UTC is equivalent to GMT, however their measurements differ: GMT is measured from midday, whereas UTC is measured from midnight. UTC is the legal basis of time in the world.

@janewang Sure, we can use the UTC timezone. The design showed GMT.

File: src/helpers/formatEpochToDate.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  const dateTimeFormatter = new Intl.DateTimeFormat("en-US", {
+    weekday: "short",
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    second: "numeric",
+    hourCycle: "h24",
+    timeZone: "utc",
```
Comment (jeesunikim) [inline src/helpers/formatEpochToDate.ts:12]: assuming it doesn't matter whether this is capitalized or not ([mdn doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat))

**Review by jeesunikim — APPROVED** [review]

File: src/helpers/formatEpochToDate.ts (hunk 1 above)
Comment (quietbits) [reply src/helpers/formatEpochToDate.ts:12]: Yes, both seem to be correct values. Thanks for checking! :raised_hands:

**Pull request merged**